### ISSUE TITLE
Change name to embrace other Arm platforms

### ISSUE THREAD

**Issue by ctrahey** [raspbernetes/multi-arch-images#140]
# Details

**Describe the solution you'd like:**

> ARM is growing! For myriad reasons, ARM is gaining traction beyond SBCs. For example, AWS Graviton instance types and the coming wave of publicly available ARM CPUs for servers. It seems to my eyes like more people are seeking multi-arch images, and I worry that the "raspberry" heritage in the name of this project will be interpreted as "not enterprise" or "not for Graviton". Is it reasonable to consider a name change (or fork+deprecate) of this project to acknowledge the broader footprint and non-Pi use-cases of ARM and multi-arch images?

**Comment by xUnholy** [comment]
@ctrahey 

This is a good point and something we'll consider, happy to have a discussion in discord and throw out some ideas.

**Comment by stuartpb** [comment]
If you rename the project to reflect that it's not just for Raspberry Pi, you might as well also change the "bernetes" part too, since these images could still be useful in other Docker / OCI systems.

**Comment by xUnholy** [comment]
It's likely we'll move the `multi-arch-images` repo to the new org `hyperarch` here https://github.com/hyperarch. This will be specific for hosting images as such that are cross-compiled for multiple architecture types and follow the same pattern of deprecation once the upstream provide the functionality.

@stuartpb @ctrahey would love your thoughts.

**Comment by ctrahey** [comment]
@xUnholy Yeah, I think that's a great name! Funny, I didn't even mention M1 in the original issue - momentum for ARM is looking really strong. Appreciate the work on this project!
